packages/graphql-elasticsearch-transformer/streaming-lambda/python_streaming_function.py

```diff
@@ -18,16 +18,15 @@
 # The following parameters are required to configure the ES cluster
 ES_ENDPOINT = os.environ['ES_ENDPOINT']
 ES_REGION = os.environ['ES_REGION']
-DEBUG = True if os.environ['DEBUG'] is not None else False
+DEBUG = True if os.environ.get('DEBUG') not in ['0', 'False', 'false', None] else False
```

Comment:
It might be simpler to check if DEBUG == 1 as that is the only case in which debug logs would appear.

Reply:
You are right, fixed